pkg/controller/start.go

```diff
@@ -42,7 +42,7 @@ func Start(cfg config.Controller) error {
 	ctrl.SetLogger(zapr.NewLogger(zapLogger))
 
 	log := ctrl.Log.WithName("setup")
-	log.V(1).Info("Using provided configuration", "config", cfg)
+	log.Info("Using provided configuration", "config", cfg)
```

Comment:
This doesn't have secrets in it?

Reply:
mmm, the amqp url 😬 . i'll have to figure out how logr serializes the object and make sure that field and any others get sanitized. seems like a worthwhile effort.

Reply:
We can just implement `MarshalJSON` for any of the configuration structs whenever we want to _sanitize_ rendered data.